bindings/rust/bench/benches/throughput.rs

```diff
@@ -24,10 +24,7 @@ pub fn bench_throughput_cipher_suite(c: &mut Criterion) {
             b.iter_batched_ref(
                 || {
                     let mut harness = T::new(
-                        CryptoConfig {
-                            cipher_suite,
-                            ec_group: Default::default(),
-                        },
+                        CryptoConfig::new(cipher_suite, Default::default(), Default::default()),
```

Comment:
We could, but a lot of other places also use Default::default() as well, so if we change it here, then we should be consistent and do it everywhere. I'm not opposed to it, since it's just a few more imports and words.

Reply:
Sorry I missed it in earlier PRs, but I'd prefer `ConcreteType::default()`. Especially when looking at the actual text of the code (without Rust analyzer adding type annotations)